packages/openapi-v3/test/unit/controller-spec/controller-decorators/request-body-decorators/request-body.test.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {post, requestBody, getControllerSpec} from '../../../../..';
+import {expect} from '@loopback/testlab';
+import {model, property} from '@loopback/repository';
+
+describe('requestBody decorator', () => {
+  describe('@requestBody', () => {
```

Comment:
This second `describe` block seems rather redundant to me, can you remove it please?